airbyte-integrations/connectors/destination-snowflake/src/test-integration/java/io/airbyte/integrations/destination/snowflake/SnowflakeInsertDestinationAcceptanceTest.java

```diff
@@ -127,7 +127,8 @@ protected boolean implementsNamespaces() {
           final ResultSet tableInfo = connection.createStatement()
               .executeQuery(String.format("SHOW TABLES LIKE '%s' IN SCHEMA %s;", tableName, schema));
           assertTrue(tableInfo.next());
-          // check that we're creating permanent tables. DBT defaults to transient tables, which have `TRANSIENT` as the value for the `kind` column.
+          // check that we're creating permanent tables. DBT defaults to transient tables, which have
```

Comment:
This is also updated after running the format task I guess 😄

Reply:
Yes.